addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1791,6 +1791,29 @@ var SnippetsMenu = Widget.extend({
         this._checkEditorToolbarVisibilityCallback = this._checkEditorToolbarVisibility.bind(this);
         $(this.options.wysiwyg.odooEditor.document.body).on('click', this._checkEditorToolbarVisibilityCallback);
 
+        // Add tooltips on we-title elements whose text overflows and on all
+        // elements with available tooltip text. Note that the tooltips of the
+        // blocks should not be taken into account here because they have
+        // tooltips with a particular behavior (see _showSnippetTooltip).
+        this.tooltips = new Tooltip(this.el, {
+            selector: 'we-title, [title]:not(.oe_snippet)',
+            placement: 'bottom',
```

Comment:
Should I wait the 16.0 fix to hit master and this PR to be rebased before reviewing this one? As it's adding boundary and container property here?

Reply:
> It's the same.

I mean, not really, this is modifying the tooltip call, which will then be changed again by the[ forward port](https://github.com/odoo/odoo/pull/111032#issuecomment-1408364331), making the review of this one kinda useless as its diff will be changed by the incoming PR, right?
Anyway, seems like you forgot to r+ the mentioned forward port, I did it this morning, you should be able to rebase later today, let me know once done :)